packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -46,6 +49,7 @@ export type StyleXOptions = $ReadOnly<{
   treeshakeCompensation?: boolean,
   genConditionalClasses: boolean,
   unstable_moduleResolution: void | ModuleResolution,
+  aliases: any,
```

Comment:
This is the only place where `aliases` should be added. Remove the other ones from `ModuleResolution` type.

Reply:
Here's the type that you can use for `Aliases`.

```ts
type Aliases = $ReadOnly<{[string]: string}>;
```

Please replace all the `any` types.